api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -92,6 +97,22 @@ type IBMPowerVSClusterSpec struct {
 	CosInstance *CosInstance `json:"cosInstance,omitempty"`
 }
 
+// DHCPServer contains the DHCP server configurations
+type DHCPServer struct {
+	// Optional cidr for DHCP private network
+	Cidr *string `json:"cidr,omitempty"`
+
+	// Optional DNS Server for DHCP service
+	// +kubebuilder:default="1.1.1.1"
+	DNSServer *string `json:"dnsServer,omitempty"`
+
+	// Optional name of DHCP Service. Only alphanumeric characters and dashes are allowed (will be prefixed by DHCP identifier)
+	Name *string `json:"name,omitempty"`
+
+	// Indicates if SNAT will be enabled for DHCP service
+	SnatEnabled *bool `json:"snatEnabled,omitempty"`
```

Comment:
this should be true by default and you can rename this to `snat` to keep it simple as this is already boolean.

Reply:
Updated.